src/macos/mod.rs

```diff
@@ -108,7 +108,7 @@ impl PlatformManager {
                     u2f_is_keyhandle_valid(dev, &challenge, &application, key_handle)
                         .unwrap_or(false) /* no match on failure */
                 })
-                .collect::<Vec<&Vec<u8>>>();
+                .collect::<Vec<_>>();
```

Comment:
Odd that this wasn't caught by the linter.

Reply:
Just a tiny cleanup. I hate having to be so specific :)